magento2-nginx/fpm-7.0/README.md

```diff
@@ -2,6 +2,10 @@
 
 ```Dockerfile
 FROM quay.io/continuouspipe/magento2-nginx:fpm-7.0
```

Comment:
Not related to your PR, but can you remind me to go through all of these README's and update the FROM's!